build/images/base/Dockerfile.controller

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BUILD_TAG
+FROM ubuntu:22.04 as cni-binaries
```

Comment:
For antrea-controller only, a full Ubuntu is not necessary for antrea-controller. The target of this change is to reduce the image size as much as possible. For Ubuntu image, the size is several hundred of MB. But for some other image, the size is only tens of MB.
Could you try to find a smaller base image, e.g., the base for registry.k8s.io/kube-apiserver or registry.k8s.io/kube-controller-manager?

Reply:
sure would check that